### PR TITLE
CSE equivalent buffer-loads

### PR DESCRIPTION
**Pull request by chaosagent**
For winograd #1037; follow up to #1272; depends on #1303 to be useful. Different bufs can load the same stuff from the same rawbufs. The use case is Tensor.stack with the stack dimension unrolled.

### CODE REVIEW

**Comment by tinyb0t** [comment]
Changes made in `tinygrad/`:
```
------------------------------------------------------------
files                             insertions       deletions
------------------------------------------------------------
tinygrad/codegen/linearizer.py            10              10
------------------------------------------------------------
lines added in the tinygrad folder: 0
```

**Comment by geohot** [comment]
These PRs are really hard to follow. Can you show an example of what it changes in the generated code? Is this the minimal way to achieve that change?

**Comment by chaosagent** [comment]
This is on hold until I can get an actually performant Winograd kernel to be generated. I'll post a writeup on this sometime. Thanks!
